x-pack/plugins/infra/public/pages/metrics/hosts/hosts_content.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Query, TimeRange } from '@kbn/es-query';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import { i18n } from '@kbn/i18n';
+import React, { useState, useCallback } from 'react';
+import { SearchBar } from '@kbn/unified-search-plugin/public';
+import { InfraLoadingPanel } from '../../../components/loading';
+import { useMetricsDataViewContext } from './hooks/use_data_view';
+import { HostsTable } from './components/hosts_list_table';
+import { InfraClientStartDeps } from '../../../types';
+
+export const HostsContent: React.FunctionComponent = () => {
+  const {
+    services: { data },
+  } = useKibana<InfraClientStartDeps>();
+  const [dateRange, setDateRange] = useState<TimeRange>({ from: 'now-15m', to: 'now' });
+  const [query, setQuery] = useState<Query>({ query: '', language: 'kuery' });
+  const { metricsDataView } = useMetricsDataViewContext();
+  // needed to refresh the lens table when filters havent changed
+  const [searchSessionId, setSearchSessionId] = useState(data.search.session.start());
+
+  const onQuerySubmit = useCallback(
+    (payload: { dateRange: TimeRange; query?: Query }) => {
+      setDateRange(payload.dateRange);
+      if (payload.query) {
+        setQuery(payload.query);
+      }
+      setSearchSessionId(data.search.session.start());
+    },
+    [setDateRange, setQuery, data.search.session]
+  );
+
+  return (
+    <div>
+      {metricsDataView ? (
+        <>
+          <SearchBar
+            showQueryBar={true}
+            showFilterBar={false}
```

Comment:
Any reason to be hiding the filter bar?

Reply:
Supporting filtering is a bit more complicated so I just hit it for now to do in a follow up